mmv1/third_party/terraform/website/docs/guides/version_5_upgrade.html.markdown

```diff
@@ -374,6 +374,42 @@ proposed value to configuration (below) or apply `lifecycle.ignore_changes` to t
 +    }
 ```
 
+### Cluster deletion now prevented by default with `deletion_protection`
+
+The field `deletion_protection` has been added with a default of `true`. This field prevents
+Terraform from destroying the resource or recreating it.
+**`deletion_protection` does NOT prevent deletion outside of Terraform.**
+To destroy a `google_container_cluster`, this field must be explicitly set to `false`.
+
+### `networking_mode` defaults to `VPC_NATIVE` for newly created clusters
```

Comment:
I found this section a bit hard to parse but I'm not sure how to improve it; the change looks complex.

Reply:
I'll remove this section for now, as it has not been approved yet.